proknow-sdk/Requestor.cs

```diff
@@ -17,21 +17,23 @@ namespace ProKnow
     public class Requestor
     {
         // HttpClient is intended to be instantiated once per application, rather than per-use
-        private static readonly HttpClient _httpClient = new HttpClient(new HttpClientHandler()
+        internal static HttpClient HttpClient = new HttpClient(new HttpClientHandler()
         {
             AutomaticDecompression = DecompressionMethods.GZip | DecompressionMethods.Deflate
         });
 
         private readonly string _baseUrl;
         private readonly AuthenticationHeaderValue _authenticationHeaderValue;
+        private readonly ClientInfo _clientInfo;
```

Comment:
Do you think it would be smarter to take a `IList<KeyValuePair>` of header values that should be sent with each request?